metro.config.js

```diff
@@ -17,17 +17,21 @@ gutenbergMetroConfigCopy.resolver.extraNodeModules = new Proxy(
 				return gutenbergFolder;
 			}
 
-			// let's try find the module in the Jetpack submodule. We'll try the .pnpm folder.
+			// If not exists, let's try find the module in the Jetpack submodule. We'll try the .pnpm folder.
 			const moduleFolderPnpm = path.join(
 				process.cwd(),
 				`./jetpack/node_modules/.pnpm/node_modules/${ name }`
 			);
 
-			// pnpm uses symlinks so, let's find the target
-			const symlinkTarget = fs.readlinkSync( moduleFolderPnpm );
+			if ( fs.existsSync( moduleFolderPnpm ) ) {
```

Comment:
I noticed that the dependencies from the `package.json` file of Gutenberg Mobile were being resolved using the Jetpack path, so I added this condition to prevent it.

Reply:
Oh, is that related to the Sentry work here, or are you referring to "normal" dependencies? If the latter, perhaps we should bring that fix in its own PR and merge it sooner?

Reply:
I'm referring to normal dependencies (the one we have [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/9bf53198cbeb09ee6aca79b7cc7f1242ec702a68/package.json#L107-L109)).

> If the latter, perhaps we should bring that fix in its own PR and merge it sooner?
Sure! I'll open a new PR for this 👍 .

Reply:
Here is [the PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3659) where I'm addressing this issue.